tools/virtual_hardware.sh

```diff
@@ -49,11 +49,9 @@ function try_destroy_zpools {
     for ZPOOL_TYPE in "${ZPOOL_TYPES[@]}"; do
         readarray -t ZPOOLS < <(zfs list -d 0 -o name | grep "^$ZPOOL_TYPE")
         for ZPOOL in "${ZPOOLS[@]}"; do
-            VDEV_FILE="${VDEV_DIR:-/var/tmp}/$VDEV"
             zfs destroy -r "$ZPOOL" && \
                     (zfs unmount "$ZPOOL" || true) && \
                     zpool destroy "$ZPOOL" && \
```

Comment:
You need to replace the `&&` with`||` here, otherwise you will always print
the `warn` message.

Reply:
Done